opae.cfg

```diff
@@ -268,6 +268,126 @@
+
+      "devices": [
+        { "name": "n5013_pf", "id": [ "0x1c2c", "0x1002", "0", "0" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "n5013_pf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n5010.so",
+            "devices": [
+              { "device": "n5013_pf", "feature_id": "0xe" },
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n5013_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": [],
+              "monitor-seu": true
+            }
+          }
+        ],
+        "rsu": [
+          {
+            "enabled": true,
+            "devices": [ "n5013_pf" ]
+          }
+        ],
+        "fpgareg": [],
+        "opae.io": [
+          {
+            "enabled": true,
+            "devices": [ "n5013_pf" ]
+          }
+        ]
+      }
+    },
+
+    "n5014": {
+      "enabled": true,
+      "platform": "Silicom FPGA SmartNIC N5014",
+
+      "devices": [
+        { "name": "n5014_pf", "id": [ "0x1c2c", "0x1003", "0", "0" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "n5014_pf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n5010.so",
+            "devices": [
+              { "device": "n5014_pf", "feature_id": "0xe" },
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n5014_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": [],
+              "monitor-seu": true
```

Comment:
Is SEU supported via the FME error registers?  If not, then let's set this instance and the above instance to false.